doc/50_IDE.md

```diff
@@ -15,11 +15,11 @@ emphasizes diversity.
 
 ## Overview
 
-When first open Theia, you will see a welcome screen that quickly gets you started.
+When you open Theia for the first time, a welcome screen shows up to quickly get you started.
```

Comment:
Sounds funny to me, just because I'm too used to the "shows up" as in "he shows up to work"

Reply:
fixing to _is shown_